### DISCUSSION TITLE
Css @import error in Webpack 5, sass-loader includePaths not work.

### DISCUSSION THREAD

**Discussion by lifeilu** [webpack/webpack#14243] [Q&A]
When I upgrade the webpack configuration of the project, i encountered a problem of css @import,

the error description as follows,
`
ERROR in ./src/.../components/xxx.vue (./node_modules/css-loader/dist/cjs.js!./node_modules/vue-loader/lib/style-compiler/index.js?{"optionsId":"0","vue":true,"id":"data-v-d3ee1126","scoped":true,"sourceMap":true}!./node_modules/sass-loader/dist/cjs.js?{"sassOptions":{"indentedSyntax":true,"indentWidth":4,"includePaths":["...\\src\\styles","...\\src\\styles\\mofang","...\\src\\styles\\mofang-bc","...\\src\\styles\\themes","...\\src\\assets\\vendors"]}}!./node_modules/vue-loader/lib/selector.js?type=styles&index=0!./src/.../components/xxx.vue)
Module build failed (from ./node_modules/sass-loader/dist/cjs.js):

@import "mofang-bc/variables";
^
      File to import not found or unreadable: mofang-bc/variables.
      in /src/.../components/xxx.vue (line 6, column 1)
`

I have checked the relevant configuration, and it seems that the sass-loader includePaths not work,
the sass-loader.conf.js as follows,
`
module.exports = {
  loader: 'sass-loader',
  options: {
    sassOptions: {
      indentedSyntax: true,
      indentWidth: 4,
      includePaths: [
        utils.dir('style'),
        utils.dir('style', 'mofang'),
        utils.dir('style', 'mofang-bc'),
        utils.dir('style', 'themes'),
        utils.dir('assets', 'vendors'),
      ],
    },
  },
};
`

the dependencies of my package.json as follows,
`
"webpack": "^5.43.0",
"sass-loader": "^7.3.1",
"node-sass": "^4.14.1",
`

Does webpack know that the path in my code is not a relative path but a module that needs to be found in the includePaths of sass-loader.conf? Do I need additional configuration to make the css modules parse correctly?

Hope to get ur help, thanks a lot

**Comment by alexander-akait** [comment]
Update `sass-loader` to the latest stable version

**Reply by lifeilu** [reply]
It works for me. Thanks a lot.

by the way, the new dependencies of my package.json as follows,
`
"webpack": "^5.43.0", "sass-loader": "^12.1.0", "node-sass": "^6.0.1",
`
